apis/management.cattle.io/v3/schema/schema.go

```diff
@@ -440,6 +440,18 @@ func authnTypes(schemas *types.Schemas) *types.Schemas {
 			schema.CollectionMethods = []string{}
 			schema.ResourceMethods = []string{http.MethodGet, http.MethodPut}
 		}).
+		MustImportAndCustomize(&Version, v3.KeyCloakConfig{}, func(schema *types.Schema) {
+			schema.BaseType = "authConfig"
+			schema.ResourceActions = map[string]types.Action{
+				"disable": {},
+				"testAndEnable": {
```

Comment:
can we extract setting the test action to its own method? I see lines 446-449 being repeated several times.